### PR TITLE
Add ImageRender class for Flyte Decks

### PR DESCRIPTION
**Pull request by peridotml**
# TL;DR
This PR adds a new renderer that makes it easy to save image files to a Flyte Deck. It works with both `PIL.Image.Image` and various `FlyteFile` types.

Also slight cosmetic change on using `import pandas as pd`.

## Type
 - [ ] Bug Fix
 - [x] Feature
 - [ ] Plugin

## Are all requirements met?

 - [ ] Code completed
 - [ ] Smoke tested
 - [x] Unit tests added
 - [ ] Code documentation added
 - [ ] Any pending items have an associated Issue

## Complete description
 _How did you fix the bug, make the feature etc. Link to any design docs etc_

## Tracking Issue
https://github.com/flyteorg/flyte/issues/<number>

## Follow-up issue
_NA_
OR
_https://github.com/flyteorg/flyte/issues/<number>_

### CODE REVIEW

**Review by zeryx — APPROVED** [review]
Test passes, regarding the pandas -> pd change. It does look like this is the only place in the codebase that references pandas itself - would be ok with signing off on this minor cosmetic change.

**Review by pingsutw — APPROVED** [review]

**Review by eapolinario — APPROVED** [review]
